### PR TITLE
Support physical address

### PR DESCRIPTION
**Pull request by glebpom**

### CODE REVIEW

File: src/adapter.rs [hunk 1]
```diff
@@ -81,19 +86,25 @@ unsafe fn get_adapter(adapter_addresses_ptr: PIP_ADAPTER_ADDRESSES) -> Result<Ad
...
         friendly_name: friendly_name,
+        physical_address: physical_address,
     })
 }
 
 unsafe fn socket_address_to_ipaddr(socket_address: &SOCKET_ADDRESS) -> IpAddr {
     let sockaddr = socket2::SockAddr::from_raw_parts(std::mem::transmute(socket_address.lpSockaddr), socket_address.iSockaddrLength);
 
-    // Could be either ipv4 or ipv6
+// Could be either ipv4 or ipv6
```
Comment (liranringel) [inline src/adapter.rs]: I guess it's a mistake? The indentation was fine.

**Comment by liranringel** [comment]
FYI, I have a plan to do a breaking change, returning slices instead of `Vec`s, `&str`s instead of `String`s.
I guess you submit PRs because you use this crate. Do you prefer that I'll make a release with your changes before the breaking changes? The break is small anyways but I don't mind.

File: src/adapter.rs (hunk 1 above)
Comment (glebpom) [reply src/adapter.rs]: Fixed

**Comment by glebpom** [comment]
@liranringel yes, I think it makes sense to release this changes at first. BTW, are you going to bump minor version with this incompatible changes?

**Comment by liranringel** [comment]
Yes, don't worry :)
I'm aware to semver.

**Pull request merged**